src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Query/PropertyPages/UIPropertyEditorDataProducer.cs

```diff
@@ -72,7 +72,7 @@ IEnumerable<IEntityValue> createEditorValues()
                     {
                         yield return CreateEditorValue(queryExecutionContext, parent, new ValueEditor { EditorType = "FilePath" }, properties);
                     }
-                    else if (stringProperty.Subtype == "directory")
+                    else if (stringProperty.Subtype == "folder" || stringProperty.Subtype == "directory")
```

Comment:
Don't we normally explicitly specify the type of string comparison?

Reply:
Thanks, I added one and made it case-insensitive.